.gitlab-ci.yml

```diff
@@ -281,3 +315,43 @@ notify-prod-stable-failure:
   script:
     - 'MESSAGE_TEXT=":host-red: $CI_PROJECT_NAME release pipeline <$BUILD_URL|$COMMIT_MESSAGE> failed."'
     - postmessage "#rum-deploy" "$MESSAGE_TEXT"
+
+########################################################################################################################
+# Scheduled tasks
+########################################################################################################################
+
+staging-reset-scheduled:
+  stage: task
+  only:
+    variables:
+      - $TARGET_TASK_NAME == "staging-reset-scheduled"
+  tags: ['runner:main', 'size:large']
+  image: $CI_IMAGE
+  before_script:
+    - eval $(ssh-agent -s)
+  script:
+    - node --no-warnings scripts/staging-reset.js
+
+staging-reset-scheduled-success:
+  extends: .prepare_notification
+  only:
+    variables:
+      - $TARGET_TASK_NAME == "staging-reset-scheduled"
+  script:
+    - NEW_STAGING=`date +"staging-%U"`
+    - 'MESSAGE_TEXT=":white_check_mark: [*browser-sdk*] Staging has been reset from *${CURRENT_STAGING}* to *${NEW_STAGING}*."'
+    - postmessage "#rum-deploy" "$MESSAGE_TEXT"
```

Comment:
unrelated note: it could be nice to rename `#rum-deploy` to `#browser-sdk-deploy` at some point

Reply:
Yes, sure. I will do it in a following PR.